### ISSUE TITLE
In scan mode, button aria label is not read if it is preceeded by a fluent ui dropdown control.

### ISSUE THREAD

**Issue by vasanthkumar258** [microsoft/fluentui#14166]
### Environment Information

- **Package version(s)**: 7.107.5
- **Browser and OS versions**: Microsoft Edge - 84.0.522.40 and Windows 10

### Describe the issue:

There is a fluent ui dropdown control followed by a button which has aria-label. Narrator is reading the button aria label while tabbing. But in scan mode, when using 'down arrow' key, button aria label is not read.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/vasanthkumar258/pen/abdXqdB?editable=true%3Dhttps%3A%2F%2Fdeveloper.microsoft.com%2Fen-us%2Ffluentui&__cf_chl_jschl_tk__=a4757e733814250add5885d0051b76572082e5a2-1595429222-0-AZJ481gqzKOMzewjH968bRrBiKIJkkXhbd3pUju5obBdFNnwk2Ojhbxpwzk3jaIytrYb5gUE9-dUK5Nk8TEUH8QxQbPQ0rY_KXqv_e7y2xlNZJ4pjbW9R5n7yil_KuBKFN5umKASWEm3IgE4eNETXQb7_m76WfnioPZ3uoylYs-14GUum0QLRrj7uFEaJhkWIFCFuJ1-wBud-zD4rbi47g090RGmrnc4qBphVGhZMQV_OZ0_Ay5TMwXZrdoe-0HLedWxcuz1moTTA0FaBAzXGPfjPdk_h56Rb1kUD36eR8ZV7MgnShfZgz6WQYu0PBhbCj_GSxLJtN_faoZoX9NGsQgIQqdhTNKQNz_IvN7PbtA-7dd7_dV1MDbUwsoblIRfSLAICMY1K0KFzkl4RNmGyHO8T7nD0F-REqHYui2a_gunoqGRNN8RHhulV6zgmmq1xsa_VT8TvMIOgXTNx9DWnXo

#### Actual behavior:

When navigating in scan mode, narrator is reading the dropdown values but when we navigate to button using "down arrow", button aria-label is not read

#### Expected behavior:

Button aria-label should be read in scan mode.

**Comment by paulgildea** [comment]
@smhigley FYI

**Comment by xugao** [comment]
@vasanthkumar258 - I can't repro using your [codepen](https://codepen.io/xugao/pen/LYGqBYJ) using Edge(Version 84.0.522.40 (Official build) (64-bit)) and Narrator. I can hear the aria label being read when arrow down from dropdown in scan mode

**Comment by vasanthkumar258** [comment]
@xugao , 
Steps to repo:
1. Open the codepen mentioned in the issue description.
2. Navigate in scan mode from dropdown, do not select anything in the dropdown or do not open the dropdown listbox. 
3. Use "down arrow" to button, we can see that focus does not stay on the button, aria-label is not read and immediately focus is shifted to whole screen.
4. Now if we use "up arrow", button is properly focussed and aria-label is read out.

Please look at the zip attachment which contains mp4 video with issue. 

[Media2.zip](https://github.com/microsoft/fluentui/files/4965680/Media2.zip)

**Comment by xugao** [comment]
@vasanthkumar258 - I believe this is a Narrator bug. I filed a bug [here](https://microsoft.visualstudio.com/OS/_workitems/edit/27887402)

**Comment by xugao** [comment]
Closing as this is Narrator. feel free to re-open if you need further help from us

**Comment by vasanthkumar258** [comment]
@xugao  That issue has been closed as not repro. Can you please check this issue from your end.
